packages/nimble-components/src/wafer-map/styles.ts

```diff
@@ -1,3 +1,50 @@
 import { css } from '@microsoft/fast-element';
+import {
+    borderColor, borderWidth
+} from '../theme-provider/design-tokens';
 
-export const styles = css``;
+export const styles = css`
```

Comment:
Modified the structure as suggested. However, the .top/.bottom/.left/.right is an extension of the the svg-root class.
So currently the structure looks something like this:
.wafer-map-container {}
.svg-root {}
.top {}
.right {}
.left {}
.bottom {}
.zoom-container {}
.circle-base {}
.circle-drawing-path {}
.wafer-map-area {}

Also, every child element of the .wafer-map-container should have the same size as the container. Because of this I left the 500px width and height only for the .wafer-map-container and used "width: inherit" and "height: inherit" for all the other child elements.

Reply:
Looks great! I added more explanation for the syyntax I was proposing for .svg-root.top`, etc. below. I believe the syntax I show in my comment should work as expected.

Reply:
Yeah, I was a bit confused as `${x => x.orientation}` provides only the string "top"/"bottom"/"left"/"right" and wasn't sure how it is going to behave, but I tested it and everything works fine. I went and implemented as suggested by you,